.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md) for documentation of individual tasks.  
+
+## Tasks
+
+**Notes**:
+* Weeks are only specified to give some orientation.
+* The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.
+
+Week -3 to 1:
+* [ ] [Release Lead] Set a tentative release date for the minor release
+* [ ] [Release Lead] Assemble release team
+
+Week 1:
+* [ ] [Release Lead] Finalize release schedule and team
+* [ ] [Release Lead] Prepare main branch for development of the new release
+* [ ] [Release Lead] Create a new GitHub milestone
+* [ ] [Communications Manager] Add docs to collect release notes for users and migration notes for provider implementers
+* [ ] [Communications Manager] Update supported versions
+
+Week 1 to 4:
+* [ ] [Release Lead] [Track] Remove previously deprecated code
+
+Week 6:
+* [ ] [Release Lead] Cut the v1.3.1 release
+
+Week 9:
+* [ ] [Release Lead] Cut the v1.3.2 release
+
+Week 11 to 12:
+* [ ] [Release Lead] [Track] Bump dependencies
+
+Week 13:
+* [ ] [Release Lead] Cut the v1.4.0-beta.0 release
+* [ ] [Release Lead] Cut the v1.3.3 release
+
+Week 14:
+* [ ] [Release Lead] Cut the v1.4.0-beta.1 release
+* [ ] [Release Lead] Select release lead for the next release cycle
+
+Week 15:
+* [ ] [Release Lead] Create the release-1.4 release branch
+* [ ] [Release Lead] Cut the v1.4.0-rc.0 release
+* [ ] [CI Manager] Setup jobs and dashboards for the release-1.4 release branch
+* [ ] [Communications Manager] Ensure the book for the new release is available
+
+Week 15 to 17:
+* [ ] [Communications Manager] Polish release notes
+
+Week 16:
+* [ ] [Release Lead] Cut the v1.4.0-rc.1 release
```

Comment:
Should we have some sort of go/no-go task for CI manager to ensure tests are passing on release branch before cutting the .0?

Reply:
Sorry what is a go/no-go tasks? I'm not familiar with the expression. 

Essentially release lead is checking in with CI manager if CI is stable enough to cut the release?

Reply:
Yeah kind of like a checkpoint to make sure release lead checks with CI manager that everything is in place before cutting the release

k8s has https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#release-cutting---go-or-no-go

Reply:
Hm sounds reasonable, although I would assume that the release lead is aware of current release blocking issues and that CI failures are surfaced as release blocking issues + CI Manager and release lead are talking to each other anyway.

I wonder how to model this here.

Some questions:
* For every release or only RC + .0?
* Should this be a separate task before every "Cut the v1.x.y.z release" task, those would be a lot?
* Should we assume that in general the release lead is up-to-date through issues + staying on top of release-blocking issues anyway so we would just add an additional task at the beginning of "Cut a release" like "Check in with the CI manager if CI is stable enough to release".

I would try to keep this as lightweight as we can.

Reply:
Added a note for it as first step of cutting a release